completions/bash/kpod

```diff
@@ -140,6 +140,15 @@ _kpod_save() {
      _complete_ "$options_with_args" "$boolean_options"
 }
 
+_kpod_export() {
+     local options_with_args="
+     --output -o
```

Comment:
IDK for sure, but is the -o ok here?  The buildah variant of this file has just the fully spelled out options.